<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="OutputImage" name="widgetOutput" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbOpenMask">
       <property name="text">
        <string>Open Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbShowMask">
       <property name="text">
        <string>Show Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMask">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOpenImages">
       <property name="text">
        <string>Open Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listViewInputs">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Save directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditSaveDir">
         <property name="text">
          <string>data/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbStart">
       <property name="text">
        <string>Start Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listViewOutputs">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarProcess">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="chbUseInvert">
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chbUseMask">
       <property name="text">
        <string>Use Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="chbUseRect">
         <property name="text">
          <string>Use Rect</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="dsbAngleStart">
         <property name="maximum">
          <double>9999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QSpinBox" name="sbXRect">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>1178</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QSpinBox" name="sbHRect">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="chbUseNeededWidth">
         <property name="text">
          <string>Use Width Rectangle
for output result</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chbUseFilter">
         <property name="text">
          <string>Use Filter</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="sbIterFilter">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="dsbAngleEnd">
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="value">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbMaximum">
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Default Median</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="chbUseGaussian">
         <property name="text">
          <string>Use Gaussian Filter</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="sbNeededWidth">
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>540</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QSpinBox" name="sbWRect">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>540</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Filter Kernel size</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QSpinBox" name="sbYRect">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Angle Range</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="sbKernelSize">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="chbUseThreshold">
         <property name="text">
          <string>Use threshold (0-1)</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="dsbTresh">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OutputImage</class>
   <extends>QWidget</extends>
   <header>outputimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
